src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -126,6 +126,10 @@ public function configure(Event $event = null)
                     $output = $output->getErrorOutput();
                 }
                 $this->exceptionHandler = function ($e) use ($app, $output) {
+                    if (!$e instanceof \Exception) {
+                        $e = new FatalThrowableError($e);
```

Comment:
/cc @fancyweb FYI

Reply:
👍 Might sounds useless but the function could be static + type hinted `\Throwable $e` to be consistent with what I did in https://github.com/symfony/symfony/pull/33038.